src/pages/home/report/ReportActionItem.js

```diff
@@ -150,6 +185,17 @@ class ReportActionItem extends Component {
                     />
                 );
         }
+
+        const interpolatedBackgroundColor = this.animatedBackgroundColor.interpolate(
+            {
+                inputRange: [0, 1],
+                outputRange: [colors.white, colors.honeydew],
```

Comment:
You decided not to use opacity here?

Reply:
yes, as per this [comment](https://github.com/Expensify/Expensify/issues/147480#issuecomment-1137457225) i'm using the exact hex value of a very light green color which does not include opacity.

Reply:
P.S animating with opacity is weird. if i give a style with 12% opacity then during the animation as it animated to 100% opacity of white the colors look weird. I'm glad i got a css without opacity to keep things simple here.